electrumsv/network.py

```diff
@@ -1130,7 +1130,8 @@ def _read_config(self):
                     logger.exception(e)
                     logger.error(f'fetching transaction {tx_hash}: {e}')
                 else:
-                    wallet.add_transaction(tx_hash, tx, TxFlags.StateCleared)
+                    wallet.add_transaction(tx_hash, tx, TxFlags.StateCleared |
```

Comment:
Our standards are 120 character width for code. If you get rid of the comment or move it to a separate line, this code should be able to be one line possibly.